src/gl/CMakeLists.txt

```diff
@@ -10,6 +10,8 @@ set(REALSENSE_GL_CPP
     synthetic-stream-gl.cpp
     yuy2rgb-gl.h
     yuy2rgb-gl.cpp
+    y411-2rgb-gl.h
```

Comment:
should be `y411torgb`, `y4112rgb`, but not `y411-2rgb`

Reply:
done